pkgs/cli/src/main.zig

```diff
@@ -335,9 +337,13 @@ pub fn main() !void {
             };
             defer start_options.deinit(allocator);
 
-            try node.loadGenesisConfig(allocator, custom_genesis, &start_options);
+            try node.loadGenesisConfig(allocator, custom_genesis, leancmd.override_genesis_time, &start_options);
```

Comment:
make this a boolean flag and if true pass time.now here to laodgenesisconfig else null

Reply:
That will lead to different genesis time for different nodes, I think it doesn't make sense.

Reply:
hmm make sense that way